package.json

```diff
@@ -12,13 +12,22 @@
     "eslint:strict": "STRICT=1 eslint -c src/.eslintrc.js 'src/**/*.ts?(x)'",
     "prettier": "prettier --write",
     "preview": "vite preview",
-    "start": "vite"
+    "start": "vite",
+    "test:exact": "jest --",
```

Comment:
It looks like the [syntax](https://jestjs.io/docs/cli#running-from-the-command-line) for running a specific test is just `jest <path/to/test>`. As such, I think we can use `yarn test <path/to/test>` to run a specific test, no `test:exact` needed — I added another test file and tried locally and this seems to work!